client/src/components/Shared/GreenButton/index.js

```diff
@@ -1,6 +1,9 @@
 import React from 'react';
 import StyledButton from './greenbutton.style';
 
-const GButton = () => (<StyledButton type="button">CONTINUE</StyledButton>
+const GButton = props => (
+  <StyledButton onClick={props.onClick} type="button">
+    {props.title}
```

Comment:
You could destructure here but not necessary.

Reply:
> You could destructure here but not necessary.

No you can't?